main.go

```diff
@@ -0,0 +1,30 @@
+/*
+ *Author: Stefan
+ *Date: 11/21/2019
+ *Last changes: 11/20/2019 13.12
+ *Task: Implement average([]int) float64 function that returns an average value of slice (sum / N)
+**/
+
+package main
+
+import "fmt"
+
+func average(sl []int) float64 {
+	if len(sl) == 0 {
```

Comment:
I think this lesson is done.
just in case you want to get one step further.
it's usual for go to have a signature like 
```go
func average(a []int) (float64, err){}
```